scripts/setup-centos8.sh

```diff
@@ -189,14 +207,26 @@ function install_velox_deps {
 
 (
   if [[ $# -ne 0 ]]; then
+    # Activate gcc9; enable errors on unset variables afterwards.
+    source /opt/rh/gcc-toolset-9/enable || exit 1
+    set -u
     for cmd in "$@"; do
       run_and_time "${cmd}"
     done
+    echo "All specified dependencies installed!"
   else
+    if [ "${INSTALL_PREREQUISITES:-Y}" == "Y" ]; then
+      echo "Installing build dependencies"
+      run_and_time install_build_prerequisites
+    else
+      echo "Skipping installation of build dependencies since INSTALL_PREREQUISITES is not set"
+    fi
+    # Activate gcc9; enable errors on unset variables afterwards.
+    source /opt/rh/gcc-toolset-9/enable || exit 1
```

Comment:
Do we need to do this twice ?

Reply:
One is in the `if` branch, and the other is in the `else` branch.

Reply:
nit: maybe move it out then...

Reply:
Can't move out. In the if branch, we can only enable this after installing build dependencies.

Reply:
The if branch must be only taken if the build dependencies are already installed. The user is expecting to build specific packages.